tests/draft2019-09/additionalProperties.json

```diff
@@ -118,14 +118,14 @@
         "description": "additionalProperties should not look in applicators",
         "schema": {
             "allOf": [
-                {"properties": {"foo": {}}}
+                { "properties": { "foo": {} } }
```

Comment:
If possible in general, good to try to separate these out (stylistic changes) -- they add noise to the review. I'm OK in general with merging style changes (that bring localized areas in line with style used throughout) being merged even *without* review if you know they're correct -- but yeah if you're going to submit them would be great to separate them from the rest of the changes -- if it were just these I'd say "lgtm" immediately and focus on the rest.

Reply:
Done.